web-components/src/components/combobox/ComboBox.ts

```diff
@@ -149,6 +150,16 @@ export namespace ComboBox {
     private multiSelected: number[] = [];
     private customContent: Element[] = [];
 
+    private notifyListItemsCountToParent() {
+      this.dispatchEvent(
+        new CustomEvent('listItemsCount-changed', {
```

Comment:
Do not use camel case for event name. We can use only small case.

Reply:
Name of this event is also not looking good. can we make it like item-count-changed